aiobotocore/endpoint.py

```diff
@@ -13,7 +13,6 @@
     logger,
 )
 from botocore.hooks import first_non_none_response
-from urllib3.response import HTTPHeaderDict
```

Comment:
That removes our only direct urllib3 dependency and it could be dropped from pyproject.toml

Reply:
done

Reply:
ya, that should have been an error, hmmm